core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -51,7 +54,9 @@ class Analyzer(
   private var settings: Settings = _
   private var reporter: PresentationReporter = _
 
-  protected var scalaCompiler: RichCompilerControl = _
+  protected var scalaCompiler: Option[RichCompilerControl] = _
```

Comment:
yeah, I remember fixing that so it would work for scala compiler / library hackers. I hope I didn't break it again.